extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -46,6 +47,12 @@ class AmpAccordion extends AMP.BaseElement {
 
     /** @private {Element} */
     this.sections_ = null;
+
+    /** @private {?../../../src/service/action-impl.ActionService} */
+    this.action_ = null;
+
+    /** @private {!Object} */
+    this.shouldHandleClickCache_ = map();
```

Comment:
sorry, I only meant caching if we don't end up using actionService. No need for double caching.